GVRf/Framework/backend_oculus/build.gradle

```diff
@@ -0,0 +1,216 @@
+
+            def oculusDir = rootProject.property("OVR_MOBILE_SDK")
+            copy {
+                from oculusDir+'/VrApi/Libs/Android/VrApi.jar'
+                into 'src/main/libs'
+            }
+            copy {
+                from oculusDir+'/VrApi/Libs/Android/armeabi-v7a/libvrapi.so'
+                into 'src/main/libs/armeabi-v7a'
+            }
+            copy {
+                from oculusDir+'/VrAppSupport/SystemUtils/Libs/Android/SystemUtils.jar'
+                into 'src/main/libs'
+            }
+        } else {
+            println "WARNING: not copying Oculus files; OVR_MOBILE_SDK not found"
+        }
+    }
+
+    task buildNative(type: Exec) {
+        if (rootProject.hasProperty("OVR_MOBILE_SDK")) {
+            environment 'OVR_MOBILE_SDK', rootProject.property("OVR_MOBILE_SDK")
+        } else {
+            println "WARNING: relying on hard-coded paths and environment variables; OVR_MOBILE_SDK not found"
+        }
+
+        def ndkbuild = ""
+        if (rootProject.hasProperty("ANDROID_NDK_HOME")) {
+            ndkbuild = rootProject.property("ANDROID_NDK_HOME")
+            ndkbuild += '/'
+        }
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            ndkbuild += 'ndk-build.cmd'
+        } else {
+            ndkbuild += 'ndk-build'
+        }
+
+        if (rootProject.hasProperty("OVR_MOBILE_SDK")) {
+            environment 'OVR_MOBILE_SDK', rootProject.property("OVR_MOBILE_SDK")
+        }
+
+        commandLine ndkbuild, '-C', file('src/main').absolutePath, '-j', Runtime.runtime.availableProcessors()//, 'NDK_DEBUG=1'
+    }
+
+    buildTypes {
+        debug {
+        }
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-project.txt'
+        }
+        releaseToGitHub {
+        }
+    }
+
+    sourceSets.main {
+        java.srcDirs = ['src/main/java']
+        jni.srcDirs = [] // no auto generation of Android.mk
+        // pre-compiled libraries
+        jniLibs {
+            srcDir 'src/main/libs'
+        }
+    }
+
+    task cleanNative(type: Exec) {
+        def ndkbuild = ""
+        if (rootProject.hasProperty("ANDROID_NDK_HOME")) {
+            ndkbuild = rootProject.property("ANDROID_NDK_HOME")
+            ndkbuild += '/'
+        }
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            ndkbuild += 'ndk-build.cmd'
+        } else {
+            ndkbuild += 'ndk-build'
+        }
+
+        if (rootProject.hasProperty("OVR_MOBILE_SDK")) {
+            environment 'OVR_MOBILE_SDK', rootProject.property("OVR_MOBILE_SDK")
+        }
+        commandLine ndkbuild, '-C', file('src/main').absolutePath, '-j', Runtime.runtime.availableProcessors(), 'clean'
+    }
+
+    clean.dependsOn 'cleanNative'
+
+    tasks.withType(JavaCompile) {
+        compileTask -> compileTask.dependsOn buildNative, copyOculusFiles
+    }
+
+    task eclipseZip(type: Zip) {

```

Comment:
Is it safe to delete this task? Seems like the directory structure and packing rules don't apply anymore.